### DISCUSSION TITLE
vue-cesium just rendering a blue globe

### DISCUSSION THREAD

**Discussion by tob-at-lovelace** [zouyaoji/vue-cesium#487] [Q&A] [answered]
Hi everyone. I absolutely love this project - thank you so much for making this happen!

I found some old sample code of using vue-cesium 2.x with Vue 2.x and Nuxt 2.x. I know that all of this is out of date, but it seemed like an OK starting point. Things are working, but I'm just getting a blue globe with no terrain. I consistently get a console error saying:

```
Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'tileXYToRectangle')
    at new $1 (Cesium.js:11308:67825)
```

I decided to take the leap tonight into updating everything, and I noticed that the demo on the vue-cesium 3.x web page has exactly the same issue: https://zouyaoji.top/vue-cesium/#/en-US/component/installation shows a plain blue globe in the Result tab of the Hello World example.

I *really* want to get all of this working together, but I'm not finding many good end-to-end examples. Thank you in advance for your help!

**Comment by zouyaoji** [comment, marked as answer]
Hello, please use the "latest" tag. The "next" tag is no longer being updated. I apologize for any inconvenience.

![image](https://github.com/zouyaoji/vue-cesium/assets/24430116/cfe59060-5a7f-43e6-8629-857b8fe31faa)

**Comment by tob-at-lovelace** [comment]
Thank you!

I'm still having this issue with my old Nuxt2 code (which doesn't pull in the .css file directly). I see there's a number of nice templates available, but I don't see a recent one with Nuxt3. Is there one that I'm missing or would you suggest I restructure things to be more like one of the existing examples?

Thanks again for your quick response.

**Reply by tob-at-lovelace** [reply]
Actually, i discovered this repo which seems to be exactly what I was hoping for.

https://github.com/nakamura196/nuxt3-vuetify-cesium

Thanks again!

**Reply by zouyaoji** [reply]
Hello, I didn't provide a template for Nuxt because I haven't worked with Nuxt before, and I'm not quite familiar with how to set it up.

If you encounter any issues using vue-cesium in Nuxt, feel free to bring them up for discussion.
